pkg/util/tracing/tracingservicepb/tracing_service.proto

```diff
@@ -20,7 +20,10 @@ message SpanRecordingRequest {
 }
 
 message SpanRecordingResponse {
-  repeated tracing.tracingpb.RecordedSpan span_recordings = 1 [(gogoproto.nullable) = false];
+  message Recording {
```

Comment:
Instead of defining this, is it possible to unify this with tracing.Recording instead? Either through gogoproto.casttype or by making tracing.Recording a pb type (I think the former option is preferable).

Reply:
As discussed offline I tried the latter, but it is a prettty big diff. Still debating if it is worth it over in https://github.com/cockroachdb/cockroach/pull/66692.

Reply:
Yea, fair, sorry for the goose chase. Leaving it as is seems fine to me.